### PR TITLE
breadcrumbs added to where to export and make an export plan

### PR DESCRIPTION
**Pull request by bobby-didcoding**
breadcrumbs added to core/base

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GRTLV-205
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Merging

- [x] This PR can be merged by reviewers.

### CODE REVIEW

File: tests/unit/core/test_wagtail_hooks.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def test_estimated_read_time_calculation(rf, domestic_homepage):
     revision = detail_page.save_revision()
     revision.publish()
 
-    expected_duration = timedelta(seconds=181)
+    expected_duration = timedelta(seconds=182)
```
Comment (hal274) [inline tests/unit/core/test_wagtail_hooks.py:211]: Just for my understanding, what is the difference between 181 and 182?

File: .circleci/config.yml [hunk 2]
```diff
@@ -147,6 +147,17 @@ jobs:
           key: v1-deps-{{ checksum "requirements_test.txt" }}
           paths:
             - 'venv'
+      - run:
```
Comment (sebs-code) [inline .circleci/config.yml:150]: Should this be here for this PR? Looks more involved than breadcrumbs?

File: tests/unit/core/test_wagtail_hooks.py (hunk 1 above)
Comment (sebs-code) [inline tests/unit/core/test_wagtail_hooks.py:211]: Why change this?

Comment (bobby-didcoding) [reply tests/unit/core/test_wagtail_hooks.py:211]: We have these estimated_read_duration methods peppered around the repo. they expect html to be read in xx seconds. Introducing/removing html such as breadcrumbs effect read time and  cause the test to fail.

File: .circleci/config.yml (hunk 2 above)
Comment (bobby-didcoding) [reply .circleci/config.yml:150]: The change is needed to make breadcrumbs work. The repo is now looking for breadcrumb.html that is located in node_modules. The file does not exist without running npm install.

File: tests/unit/core/test_wagtail_hooks.py (hunk 1 above)
Comment (bobby-didcoding) [reply tests/unit/core/test_wagtail_hooks.py:211]: We have these estimated_read_duration methods peppered around the repo. they expect html to be read in xx seconds. Introducing/removing html such as breadcrumbs effect read time and cause the test to fail.

Comment (sebs-code) [reply tests/unit/core/test_wagtail_hooks.py:211]: Gotcha - not criticising your increase (I can see why your had to do it), but the test itself seems absolutely insane to me. Will approve as it looks like the test has been here a long time.

**Review by sebs-code — APPROVED** [review]

**Pull request merged**